### ISSUE TITLE
How to deal with multi-author posts?

### ISSUE THREAD

**Issue by artlogic** [arsmentis/wdet-conversion#20]
There are roughly 600 stories in the system that have more than one author. How do you want to deal with this?

**Comment by wdet30** [comment]
Create a user named "WDET Newsroom" push all post to that. Can we add the name of the Authors to the post anyway? 

I would like to use a plugin for this.
https://wordpress.org/plugins/co-authors-plus/

Please advise on above?

**Comment by artlogic** [comment]
I would suggest going with the plugin for the import as well (assuming this is the plugin you want for sure). It's probably better to keep things consistent.

Can you create a few multi author test posts on the layout site using that plugin?

**Comment by wdet30** [comment]
I will do.

I will have that running Tuesday eve or overnight.

________________________________

**Comment by wdet30** [comment]
I was able to address this issue overnight. There are two things I'll be testing today one will be a custom taxonomy. And a plug-in. If the job takes less than 7 hours I will create the custom taxonomy and have that ready for tomorrow. If not we will go with the plugin


Tony
________________________________

**Comment by wdet30** [comment]
@artlogic, I have stup the plugin and reqest for pro version You are copied on the email. See it in action here:

Bottom of post
https://layout.wdet.org/2021/09/22/james-wins-big-at-summit-climb/

And here 

https://layout.wdet.org/2021/09/22/wordpress-co-authors-multiple-authors-and-guest-authors-in-an-author-box/

TB

**Comment by artlogic** [comment]
Here's the plugin we're using: https://publishpress.com/authors/

Here's are some example posts:
* https://layout.wdet.org/2020/11/06/between-takes-the-brief-mysterious-life-of-bassist-james-jamerson/
* https://layout.wdet.org/2021/09/23/fight-over-custer-statue-in-monroe-michigan-comes-to-a-standstill/

Assigning to myself to sort out for the import.

**Comment by artlogic** [comment]
* For each person being pulled over, create a user and an author.
  * Put profile information in both the user and author objects as it's not certain we'll stick with this plugin

**Comment by wdet30** [comment]
Yes, please go to the plugin first for the purpose of preserving the multi-author issue. We can sync users from the plugin.

**Comment by artlogic** [comment]
This work still needs to be done. Re-opening.

**Comment by wdet30** [comment]
Copy
